### DISCUSSION TITLE
Compatibility with Radium/Lithium/Canary

### DISCUSSION THREAD

**Discussion by zFERDQFREZrzfq** [CreativeMD/LittleTiles#975] [Show and tell]
I noticed collisions issues when Radium or other Lithium port is installed.
You can basically walk through any shape that is not a full block.

To fix this, the following lines need to be added in the `lithium.properties` config file **on client side** :
- mixin.shapes.specialized_shapes = false
  - Fixes collision with fixed tiles
- mixin.entity.collisions = false
  - Fixes collisions with animated tiles (door, etc.)

For those who uses Canary, it is not configurable, so you would have to replace it with Radium :)

**Comment by CreativeMD** [comment]
I have asked the author of Lithium if we can figure out a solution together:
https://github.com/CaffeineMC/lithium-fabric/issues/599
